api/db/changelog/db.changelog-4.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenstein" id="changelog-4">
+    <addColumn tableName="workspace">
+      <column name="disease_focused_research" type="boolean"/>
```

Comment:
At the risk of being extra verbose, I'd recommend adding "research_purpose_" or "rp_" or something to the front of all these columns.